src/components/tutor-schedule/TutorSchedule.tsx

```diff
@@ -0,0 +1,39 @@
+import { Avatar, Box, Button, Typography } from '@mui/material'
+import { styles } from '~/components/tutor-schedule/TutorSchedule.styles'
+import { items } from './TutorSchedule.constants'
+import MessageIcon from '@mui/icons-material/Message'
+
+function TutorSchedule() {
+  return (
+    <Box sx={styles.general_container}>
+      <Typography sx={styles.section_title}>Your schedule</Typography>
```

Comment:
you can use enum even like that

<Typography variant={TypographyVariantEnum.YOUR VARIANT}>